### PR TITLE
tfupdate: init at 0.6.7

### PR DESCRIPTION
**Pull request by abhishekbiyala**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* The top-level package collection of nixpkgs.
+/* The top-level package collection of nixpkgs./tfw
```
Comment (dylanmtaylor) [inline pkgs/top-level/all-packages.nix:1]: Why is there a /tfw here?/

Comment (abhishekbiyala) [reply pkgs/top-level/all-packages.nix]: fixed

**Comment by Alizter** [comment]
Result of `nixpkgs-review pr 222288` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tfupdate</li>
  </ul>
</details>

**Review by Alizter — APPROVED** [review]

**Review by lilyinstarlight — APPROVED** [review]
In the future, could you fill out the PR template? It helps to know how you've tested your package and what platforms you've tested it on, as well as getting a basic description of what package you're adding

Aside from that, this looks good to me though. Thank you!

**Review by drupol — CHANGES_REQUESTED** [review]
There are conflicts, please rebase your PR.

**Review by Janik-Haag — APPROVED** [review]

**Comment by Janik-Haag** [comment]
Actually some one else got a other pr merged already with out looking for this one :( 
But still thank you for your contribution and keep the coming :heart: 
I suggest you open a new PR adding your self as a maintainer for tfupdate. 
And will close this one since tfupdate is now already in nipxkgs.

**Pull request closed without merging**